crypto/nrf_cc310_platform/CHANGELOG.rst

```diff
@@ -9,6 +9,68 @@ Changelog - nrf_cc3xx_platform
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_platform - 0.9.15
+***************************
+
+New version of the library with the following features:
+
+* Added new shadow key APIs for key derivation using KMU, which is compatible with multi-part operations.
+* Minor bugfixes and improvements.
```

Comment:
```suggestion
* Minor bug fixes and improvements.
```

A point like this should be last on the list.

Reply:
Fixed and made it last item in list